src/components/VErrorImage/VErrorSection.vue

```diff
@@ -0,0 +1,23 @@
+<template>
```

Comment:
Perhaps the folder for these files should be renamed to "VErrorSection"? As it's not only about the error image.

Reply:
This was on my mind as a todo, right till I pushed the PR. Thank you so much for catching it. Renamed in 2599478.